Gems/Atom/Feature/Common/Code/Source/Mesh/MeshFeatureProcessor.cpp

```diff
@@ -123,11 +123,26 @@ namespace AZ
                     }
                 };
                 Job* executeGroupJob = aznew JobFunction<decltype(jobLambda)>(jobLambda, true, nullptr); // Auto-deletes
-                parentJob->StartAsChild(executeGroupJob);
+                if (parentJob)
+                {
+                    parentJob->StartAsChild(executeGroupJob);
+                }
+                else
+                {
+                    executeGroupJob->SetDependent(&jobCompletion);
+                    executeGroupJob->Start();
+                }
             }
             {
                 AZ_PROFILE_SCOPE(AzRender, "MeshFeatureProcessor: Simulate: WaitForChildren");
-                parentJob->WaitForChildren();
+                if (!parentJob)
```

Comment:
Minor suggestion - switch the if/else blocks to make it "if (parentJob)" so it matches the previous if/else block.

Reply:
re-ordered